doc/glossary.rst

```diff
@@ -24,6 +24,9 @@ general neuroimaging concepts. If you think a term is missing, please consider
         object class and :ref:`tut-annotations`
         for a tutorial on how to manipulate such objects.
 
+    Beamformer 
+        Beamformer is a popular source estimation approach that uses a set of spatial filters (beamformer weights) to compute time courses of sources which coordinates are predefined. Beamformer weight for a particular source is selected so it minimizes average reconstructed source power at a given location, subject to a unit gain constraint. See :class:`mne.beamformer.Beamformer’.
```

Comment:
@britta-wstnr okay for you?

Reply:
not all beamformers have the unit gain constraint. I would just remove the sentence `Beamformer weight for a  ... constraint.`